config/locales/en.yml

```diff
@@ -148,10 +148,10 @@ en:
           create_account_html: or %{create_account_link}.
           description_html: Save and apply for jobs in schools.
           link_text:
-            create_account: create an account
+            create_account: create a jobseeker account
         title: For jobseekers
       personal_statement_section:
-        description_html: You can also apply for jobs, set up job alerts and %{personal_statement_guide_link}.
+        description_html: You can apply for jobs, set up job alerts and %{personal_statement_guide_link}.
```

Comment:
Instead of linking to the personal statement guide, could we link to all the jobseeker guides here? https://teaching-vacancies.service.gov.uk/jobseeker-guides

Reply:
Done!